### ISSUE TITLE
Line breaks in XPath string literal cause 'unexpected token' problem to be reported erroneously

### ISSUE THREAD

**Issue by pgfearo** [DeltaXML/vscode-xslt-tokenizer#92]
First reported in: [XPath Notebook #6](https://github.com/DeltaXML/vscode-xpath-notebook/issues/6)

Sample XPath:
```
let $xml1 := parse-xml('<root>
  <item>a</item>
  <item>b</item>
</root>')
return $xml1/root/item
```

**Comment by pgfearo** [comment]
This issue was fixed in release VS Code 0.3.17 see [Release Notes](https://github.com/DeltaXML/vscode-xslt-tokenizer/wiki/Release-Notes)

**Comment by pgfearo** [comment]
Issue fixed in latest release
